libse/SubtitleFormats/FinalCutProXml15.cs

```diff
@@ -469,39 +475,22 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
             }
         }
 
-        private static TimeCode DecodeTime(XmlAttribute duration)
+        private static TimeCode DecodeTime(XmlAttribute xduration)
```

Comment:
Ups, this is not right - "ms" is milliseconds and "s" is seconds - also "duration" or similar is better than "xduration"...

Reply:
```c#
if (duration != null)
{
	var dur = duration.Value;
	if (dur.EndsWith("ms"))
	{
		var arr = duration.Value.TrimEnd('s').TrimEnd('m').Split('/');
		if (arr.Length == 2)
		{
			return TimeCode.FromSeconds((long.Parse(arr[0]) * 1000.0) / (double.Parse(arr[1]) * 1000.0));
		}
		if (arr.Length == 1)
		{
			return TimeCode.FromSeconds(float.Parse(arr[0]) * 1000.0);
		}
	}
```

Are we supposed to divide by **1000** to get to seconds instead?
e.g: return TimeCode.FromSeconds((long.Parse(arr[0]) **/** 1000.0) / (double.Parse(arr[1]) **/** 1000.0));

/cc @niksedk